ts-scripts/utils.ts

```diff
@@ -189,7 +189,7 @@ export function route(connectionType, buildType) {
         } else if (isSourceScript(req.url)) {
             readFile(join(__dirname, '../src', req.url), 'utf8')
                 .then((code) => {
-                    if (code.indexOf('@decorators') !== -1) {
+                    if (code.indexOf('@') !== -1) {
```

Comment:
Won't it break on the code with `@` chars in string literals?
For example, `const email = 'user@example.com';`?